commands/make/make.drush.inc

```diff
@@ -281,6 +289,31 @@ function drush_make($makefile = NULL, $build_path = NULL) {
 }
 
 /**
+ * Drush callback; convert makefile to YAML.
+ */
+function drush_make_convert_yaml($makefile = NULL) {
+
+  drush_log(dt('Beginning to convert !makefile.', array('!makefile' => $makefile)), 'ok');
+
+  $file = $makefile . '.yml';
+  $info = make_parse_info_file($makefile);
+
+  require dirname(__FILE__) . '/../core/outputformat/yaml.inc';
+  $dumper = new drush_outputformat_yaml(array());
```

Comment:
Here you probably want to use Drush's engines, instead of a 'require' and a 'new'. Something like:

$dumper = drush_load_engine('outputformat', 'yaml');